zingolib/src/commands/utils.rs

```diff
@@ -76,6 +80,22 @@ pub(super) fn parse_send_args(
     Ok(send_args)
 }
 
+/// Checks send inputs do not contain memo's to transparent addresses.
```

Comment:
If possible add a doc-test, if that's not possible a unit-test, if that's not possible an explicit TODO.

Reply:
will do

Reply:
oh this is not pub, i will add a unit test